src/features/user/Widget/EmbedModal.tsx

```diff
@@ -63,7 +64,7 @@ export default function EmbedModal({
     };
     if (contextLoaded && token) {
       try {
-        const res = await putAuthenticatedRequest(
+        const res: User = await putAuthenticatedRequest(
           `/app/profile`,
```

Comment:
`putAuthenticatedRequest` supports generics, so you can add `<User>` as the expected type of the response instead of annotating the type